wordpress-mobile/WordPress-iOS
# Async: Show uploading state in media cells

This PR adds an indeterminate progress indicator to uploading media items in the Media Library. The next PR will show the actual progress value for those items.

![async-1](https://user-images.githubusercontent.com/4780/32802170-2a6e81de-c977-11e7-8089-6c959a735f71.gif)

To test:

* Navigate to the Media Library
* Upload media items using the async debug option
* Ensure that the progress indicators appear, and disappear when an image finishes uploading.
* Ensure that the progress views work correctly with when cells are scrolled off and on the screen.
* Try navigating out of the media library and back in during uploads, to check that the correct state is displayed.
* Check that the cells are removed if uploads fail.
